lib/util/LocationStatusManager.js

```
@@ -166,10 +170,10 @@ class LocationStatusManager {
         async.eachLimit(newLocations, 10, (location, next) => {
             const locationConfig = new LocationStatus(this._supportedServices);
             this._locationStatusStore[location] = locationConfig;
-            this._locationStatusColl.insert({
+            this._locationStatusColl.insertOne({
                 _id: location,
                 value: locationConfig.getValue(),
-            }, next);
+            }).finally(next);
```

`finally` does not receive the error, this will silently ignore any error....